variables-skyline/index040.html

```diff
@@ -15,5 +15,5 @@
 </html>
 
 <!--
-  Now you can see your body, it's the hoizontal line on your page. The box around it is the html element. Make your body fill the whole viewport by giving it a `height` of `100vh`. Remove the default margin from the body by setting the `margin` to `0`. Finally, set the `overflow` property to `hidden`. This will hide any scroll bars that appear when something extends past the viewport.
+  Now you can see your body, it's the horizontal line on your page. The box around it is the html element. Make your body fill the whole viewport by giving it a `height` of `100vh`. Remove the default margin from the body by setting the `margin` to `0`. Finally, set the `overflow` property to `hidden` to hide any scroll bars that appear when something extends past the viewport.
```

Comment:
This could probably be split into at least two lessons, one for setting the height and margin, and the other to set the overflow to hidden.